### ISSUE TITLE
Virtualbox 5.0 release

### ISSUE THREAD

**Issue by theodorosploumis** [geerlingguy/drupal-vm#164]
The new Virtualbox 5.x is a reality. https://www.virtualbox.org/wiki/Downloads. Do not upgrade Virtualbox until vagrant supports virtualbox 5.x.

See related issue on vagrant project https://github.com/mitchellh/vagrant/issues/5925

**Comment by geerlingguy** [comment]
@theodorosploumis - Ha! I had a note in my todo list to add an issue to track the 5.0 list today, and you've beat me to the punch :)

I'm planning on rebuilding all the base boxes (for Ubuntu/CentOS) once the next stable version of Vagrant is released (should be this week, according to @mitchellh), and then I'll be adding a note in the next Drupal VM release version that VirtualBox 5.x and Vagrant 1.7.3 (?) are the new minimum recommended versions (and users should upgrade ASAP).

If someone wants to remain on the old versions of each, they can just leave the current base box in place (don't ever run `vagrant box update`), and everything should work fine.

And for those using VMWare Fusion or Parallels, you can disregard this issue entirely :)

**Comment by geerlingguy** [comment]
FYI the related issue is https://github.com/mitchellh/vagrant/issues/5926

**Comment by geerlingguy** [comment]
1.7.3 has been released (and is available on the [downloads](https://www.vagrantup.com/downloads.html)) page.

I will rebuild the base boxes soon, assuming Packer doesn't also need an update to be able to work with VirtualBox 5.0 :)

**Comment by geerlingguy** [comment]
nvm, now postponed on upstream issue with Packer: https://github.com/mitchellh/packer/issues/2429

**Comment by geerlingguy** [comment]
Now waiting on a v0.8.2 release of Packer, then the build should succeed: https://github.com/mitchellh/packer/releases

**Comment by covenantd** [comment]
v0.8.2 is out (actually came out literally 2 minutes after your last comment!)

**Comment by geerlingguy** [comment]
Haha, nice! So I just need to leave comments here and there to get other projects to release.

In that case, maybe I'll postpone something on Drupal 8 rc1 too :P

**Comment by covenantd** [comment]
You've got the POWER! Yes please for D8 rc1 :)

**Comment by geerlingguy** [comment]
Working on building new box versions this weekend.

**Comment by geerlingguy** [comment]
Boxes are built, waiting for the loooong upload to complete. I wish I had symmetrical traffic on my 100-down-but-only-5-up internet connection :-/

**Comment by geerlingguy** [comment]
No blocker anymore... also, I updated the 2.0.0 release notes to state that, if you're using VirtualBox, you need to update VirtualBox to 5.x and Vagrant to at least 1.7.4, before you can use the latest versions of the boxes correctly.

**Comment by geerlingguy** [comment]
All of my boxes are now up to date, tested and working with the latest releases of Vagrant, VirtualBox, and VMware Fusion: https://atlas.hashicorp.com/boxes/search?q=geerlingguy

Remember to run `vagrant box update` to update the local box for next time you rebuild Drupal VM!

**Comment by oxyc** [comment]
Hmmm I'm somewhat new with vagrant, do I need to run `vagrant destroy -f && vagrant up` or is a `vagrant reload`enough?

**Comment by geerlingguy** [comment]
@oxyc - To get the new box, best solution is to `vagrant destroy -f`, then `vagrant box update`, then `vagrant up` with VirtualBox 5.0 and Vagrant 1.7.4+ installed on your system.

**Comment by oxyc** [comment]
Thank you, I got confused when a box said it was using VBox 5 without having done a destroy. Didn't imagine they were backwards compatible as they were be rebuilt here.
